src/job.rs

```diff
@@ -193,6 +194,19 @@ impl<'roc> From<&Job<'roc>> for Command {
     fn from(job: &Job) -> Self {
         let mut command = Command::new(&job.command.tool.as_SystemTool().name.to_string());
 
+        let home = env::vars()
+            .find_map(|(var_name, var_value)| {
+                if var_name == "HOME" {
+                    Some(var_value)
+                } else {
+                    None
+                }
+            })
+            .unwrap_or_default();
```

Comment:
env::var will look up a single variable. Could we use that to conditionally add to the environment if it's present? Probably OK to not warn if HOME isn't set.

Reply:
That makes sense. I'll do it!